cypress/integration/form_test.js

```diff
@@ -0,0 +1,19 @@
+describe("testing for mvp", () => {
+    beforeEach(() => {
+      cy.visit("http://localhost:3000/pizza");
+    });
+    it("meets mvp", () => {
+      cy.get("[data-cy=name").type("Chad").should("have.value", "Chad");
+      cy.get("[data-cy=sizes]").select("medium").should("have.value", "medium");
+      cy.get("[data-cy=pepperoni]").check().should("be.checked");
+      cy.get("[data-cy=mushroom]").check().should("be.checked");
+      cy.get("[data-cy=bacon]").check().should("be.checked");
+      cy.get("[data-cy=anchovie]").check().should("be.checked");
+      cy.get("[data-cy=instructions]")
+        .type("extra cheese please")
+        .should("have.value", "extra cheese please");
+      cy.get("[data-cy=submit]").click();
+  
+      cy.get("pre").should("exist");
+    });
+  });
```

Comment:
Glad to see that you're using `data-cy`! 
I would recommend that you break this into multiple `it` blocks so that each function is only testing one thing.
Also, the `beforeEach` call isn't necessary at the moment because you only have one test. If you had multiple then it would make more sense to visit that page before each test runs.